test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1

```diff
@@ -1,258 +1,247 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 Describe "Format-Table" -Tags "CI" {
```

Comment:
Can you insert whitespace below the copyright?

Reply:
@SteveL-MSFT Seem we lost this in all test files. 😕 I believe we should fix this all in new PR.